docs/sphinx-docs/source/user/RELEASE.rst

```diff
@@ -16,30 +16,140 @@ Features
 New in Version 5.0.5
 --------------------
 This is a point release which fixes some issues reported in earlier versions
-of 5.0.x:
-
-* The issue with the optimiser not respecting parameter bounds where set has been
-  resolved by the move to a later version of the Bumps package.
-* ?
-
-There are also some new features in this version:
-
-* ?
-
-This version of SasView is built with Sasmodels 1.0.6 and Bumps 0.9.0. 
-
-New features/improvements
-^^^^^^^^^^^^^^^^^^^^^^^^^
-* ?
-
-
-Bug fixes
-^^^^^^^^^
-* ?
+of 5.0.x. A few highlights are:
+
+* The long standing issue with the Levenberg-Marquardt optimiser not respecting
+  parameter bounds has been resolved by the move to a later version of
+  the Bumps package.
+* A bug which prevented radius_effective from being updated in $P(Q)*S(Q)$
+  models when the data were resolution smeared has been fixed.
+* There has also been a technical change to the way in which the volume normalisation
```

Comment:
Each bullet should be able to stand alone from other bullets. I'd suggest, at the very least, removing the word also. A short description of the technical change could be helpful.

Reply:
I have tried to clarify this.